car_common/src/main/java/nl/flitsmeister/car_common/CarMapRenderer.kt

```diff
@@ -193,37 +189,8 @@ class CarMapRenderer(
     @Synchronized
     override fun onScroll(distanceX: Float, distanceY: Float) {
         Log.v(LOG_TAG, "onScroll distanceX($distanceX) distanceY($distanceY)")
-        val mapInstance = mapContainer.mapViewInstance ?: return
-
-        if (currentTouchPoint == null) {
-            val touchX = mapInstance.measuredWidth / 2f
-            val touchY = mapInstance.measuredHeight / 2f
-            currentTouchPoint = PointF(touchX, touchY).also {
-                mapInstance.onTouchEvent(createMotionEvent(MotionEvent.ACTION_DOWN, it.x, it.y))
-            }
-        } else {
-            currentTouchPoint?.let {
-                currentTouchPoint = PointF(it.x - distanceX, it.y - distanceY).also {
-                    mapInstance.onTouchEvent(createMotionEvent(MotionEvent.ACTION_MOVE, it.x, it.y))
-                }
-            }
-        }
-
-        resetTouchPointJob?.cancel()
-        resetTouchPointJob = runOnMainThread {
-            delay(200)
-            currentTouchPoint?.let {
-                mapContainer.mapViewInstance?.onTouchEvent(
-                    createMotionEvent(
-                        MotionEvent.ACTION_UP,
-                        it.x,
-                        it.y
-                    )
-                )
-            }
-            currentTouchPoint = null
-        }
-
+        mapContainer.moveBy(distanceX, distanceY)
+        
```

Comment:
scrollBy -> moveBy ?
That could be confusing

Reply:
Thank you for your review, I'll update this.